### PR TITLE
chore(ui-ux): organize address page components + add pagination

### PR DESCRIPTION
**Pull request by pierregee**
#### What this PR does / why we need it:
- [x] Add pagination on address page
- [x] Organize address page's components

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

**Comment by JoshuaC817** [comment]
LGTM!

File: src/pages/address/_components/WalletDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+          data-testid="wallet-transactions-title"
+          className="text-white-700 tracking-[0.01em]"
+        >
+          Transactions
+        </div>
+        <div
+          data-testid="wallet-transactions-value"
+          className="text-white-50 tracking-[0.01em]"
+        >
+          {useUnitSuffix(detail.transactionCount)}
```
Comment (lykalabrada) [inline src/pages/address/_components/WalletDetails.tsx:143]: Transaction count is 0 👀 
image

Comment (lykalabrada) [reply src/pages/address/_components/WalletDetails.tsx:143]: We can check on a separate PR. Not related to this PR anyway.

**Review by lykalabrada — APPROVED** [review]
LGTM 👍

File: src/pages/address/_components/WalletDetails.tsx (hunk 1 above)
Comment (pierregee) [reply src/pages/address/_components/WalletDetails.tsx:143]: Yea issue on  API side, weirdly, refetching it again works - assuming it is not syncing with stats properly
1st object
2nd object: refreshes the page 
image

we can investigate it more in another PR

**Pull request merged**